clayui.com/gatsby/createPages.js

```diff
@@ -128,6 +131,12 @@ module.exports = async ({actions, graphql}) => {
 		redirectInBrowser: true,
 		toPath: '/docs/css/content/typography.html',
 	});
+	actions.createRedirect({
+		fromPath: '/docs/contributing.html',
+		isPermanent: true,
+		redirectInBrowser: true,
+		toPath: CONTRIBUTING_URL,
+	});
```

Comment:
I think `createRedirect` is only used for internal redirects within the gatsby site. In the [docs](https://www.gatsbyjs.com/docs/actions/#createRedirect) it says the `toPath` should be a "URL of a created page"